### DISCUSSION TITLE
Right to left text / RTL support

### DISCUSSION THREAD

**Discussion by hshuki** [voidlabs/mosaico#634] [Q&A]
Is it possible to write text from right to left?
We are referring to languages such as Arabic and Hebrew

10X

![screenshot-2018-02-05-10 07 16](https://user-images.githubusercontent.com/36151021/35794026-a9334d1c-0a5c-11e8-9f47-03817ea07e67.png)

**Comment by bago** [comment]
I guess you can do that by altering the master template and add your "RTL" attributes where they are needed. I know there are companies using Mosaico in the RTL countries, but I don't know how they dealt with the issue, sorry.

**Comment by hadifarnoud** [comment]
the better option is to add rtl setting in Style.

**Comment by hadifarnoud** [comment]
in tinyMCE, it should be as easy as adding `ltr rtl` to here https://github.com/voidlabs/mosaico/blob/87c63738aa48c645a2e73778598ab2dc6733b457/src/js/bindings/wysiwygs.js#L245

that only fixes the text issue. I think in Styles tab, there should be an option for RTL.
